apps/gitrekt/c_src/repository.c

```diff
@@ -238,4 +238,28 @@ geef_repository_index(ErlNifEnv *env, int argc, const ERL_NIF_TERM argv[])
 	enif_release_resource(index);
 
 	return enif_make_tuple2(env, atoms.ok, term_index);
-}
\ No newline at end of file
+}
+
+ERL_NIF_TERM
+geef_repository_set_head(ErlNifEnv *env, int argc, const ERL_NIF_TERM argv[])
```

Comment:
👍

We could use that for letting the repository owner/admin change the default branch in the repository settings.